clientv3/naming/grpc.go

```diff
@@ -98,6 +98,8 @@ func (gw *gRPCWatcher) Next() ([]*naming.Update, error) {
 		case etcd.EventTypeDelete:
 			err = json.Unmarshal(e.PrevKv.Value, &jupdate)
 			jupdate.Op = naming.Delete
+		default:
```

Comment:
Looks good and it doesn't harm but curious, did you run into such case? Any empty values should be already ignored by this point. Thanks!

Reply:
Yes, it is ok not to add `default` at present, but it is more consistent with the intention from the logic point of view.

Use this pr if it helps. Thanks.

Reply:
I wanted to understand the reason behind fix and see if a test is needed. If it's more of refactoring we should update the PR message accordingly. Since empty values are already taken care by this point, default might not be added originally considering it gives impression that there can be empty values. Thank you for all your PRs.

Reply:
yes, I should modify the commit message. not "ignore empty update", It should be "ignore unknown update type' 

In combination with the situation you mentioned, it is better to use `continue` to ignore unknown type or `panic`?

thanks!

Reply:
Thanks! I don't have a strong opinion on it but using continue seems good enough as there isn't any real bug reported related to it. @hexfusion